src/test/features/claimant-response/form/models/settlementAgreement.ts

```diff
@@ -0,0 +1,47 @@
+/* Allow chai assertions which don't end in a function call, e.g. expect(thing).to.be.undefined */
+/* tslint:disable:no-unused-expression */
+
+import { expect } from 'chai'
+import { Validator } from 'class-validator'
+import { expectValidationError } from 'test/app/forms/models/validationUtils'
+import { SettlementAgreement } from 'claimant-response/form/models/settlementAgreement'
+import { ValidationErrors as GlobalValidationErrors } from 'forms/validation/validationErrors'
+
+describe('SettlementAgreement', () => {
+  describe('constructor', () => {
```

Comment:
I would advise to remove it as I think testing constructors is a bit pointless / wired.

Reply:
We are testing the constructor in all other places as well, and I suppose we are trying to make sure that the properties are initialised with the required correct default values.